core/src/test/scala/org/apache/spark/h2o/converters/DataFrameConverterTest.scala

```diff
@@ -729,6 +729,13 @@ class DataFrameConverterTest extends FunSuite with SharedH2OTestContext {
 
   }
 
+  test("Convert DataFrame to H2OFrame with dot in column name"){
+    import sqlContext.implicits._
```

Comment:
Try to test nested structure:

```
field1
field2.1
   |- field2.1.1
   |- field2.1.2
```

Reply:
good point 👍

Reply:
Implemented necessary changes together with test for this case